android/app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="ic_launcher_background">#3DDC84</color>
-</resources>
+    <color name="ic_launcher_background">#03D47C</color>
+</resources>
```

Comment:
@shawnborton please can you double-check this? The background colour we were using seemed wrong.

Reply:
Oh nice, that looks better.